core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -136,7 +136,6 @@
 import io.trino.sql.tree.Delete;
```

Comment:
Can you point out where is the name conflict? I can't see it.

Reply:
It happened in the `StatementAnalyzer` where we analyze the incoming `io.trino.sql.tree.TableArgument`, and use the `io.trino.spi.ptf.TableArgument.Builder`. I renamed the former (tree). Analogically with the descriptor arguments.